Cargo.toml

```diff
@@ -34,6 +35,7 @@ enum-iterator = "1.2.0"
 env_logger = "0.10.0"
 expect-test = "1.4.0"
 getrandom = { version = "0.2" }
+globwalk = "0.8"
```

Comment:
Is this of the 'quality' we want to depend on? Looks like it hasn't been updated in 3 years, and has open issues since about then (e.g. https://github.com/Gilnaa/globwalk/issues/29).